plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/AbstractTestDeltaLakeCreateTableStatistics.java

```diff
@@ -460,38 +455,37 @@ public void testMultiFileTable()
             assertEquals(minValues.size(), minValues.stream().distinct().count());
             assertEquals(maxValues.size(), maxValues.stream().distinct().count());
         }
+        finally {
+            assertUpdate("DROP TABLE IF EXISTS " + tableName);
```

Comment:
it's guaranteed to exist, so IF EXISTS is redundant

Reply:
I switched the logic in the tests to using `TestTable` and doing an extra `INSERT` for ensuring the existence of multiple data files.